src/frontend/AboutFile.ui

```diff
@@ -1,160 +1,156 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
![about_layout](https://user-images.githubusercontent.com/9459891/42722998-528e6710-8777-11e8-9b22-a83e6b2ec21f.png)
There is too much space on the right side.

Reply:
Ok, I'll fix that.